AUTHORING_GUIDE.md

```diff
@@ -1,207 +1,117 @@
-        much_parameter, very_function)
-
-    # Not so good
-    do_some_stuff_please(a_parameter, another_parameter, wow_so_many_parameters,
-                         much_parameter, very_function)
-    ```
-
-   Similarly with strings and other such things:
-
-    ```python
-    long_string = (
-        'Alice was beginning to get very tired of sitting by her sister on '
-        'the bank, and of having nothing to do: once or twice she had peeped '
-        'into the book her sister was reading, but it had no pictures or ...'
-    )
-    ```
-1. Use descriptive variables names in comprehensions, for example:
-
-    ```python
-    # Good
-    blogs = [blog for blog in bob_laws_law_blog]
-
-    # Not so good
-    blogs = [x for x in bob_laws_law_blog]
-    ```
-
-## The sample format
-
-In general our sample format follows ideas borrowed from
-[Literate Programming](https://en.wikipedia.org/wiki/Literate_programming).
-Notably, your sample program should be self-contained, readable from top to bottom,
-and fairly self-documenting. Prefer descriptive names. Use comments
-and docstrings only as needed to further explain. Always introduce functions and
-variables before they are used. Prefer less indirection. Prefer imperative
-programming as it is easier to understand.
-
-### Shebang
+We're happy you want to write a Python sample! Like a lot of Pythonistas,
+we're opinationed and fussy. This guide is a reference for the format and
+style expected of samples contributed to the
+[python-docs-samples](https://github.com/GoogleCloudPlatform/python-docs-samples)
+repo. The guidelines below are intended to ensure that all Python samples
+meet the following goals:
+
+* **Copy-paste-runnable.** A developer should be able to copy and paste the
+code into their own environment and run it with as few modifications as
+possible.
+* **Teach through code.** Each sample should demonstrate best practices for
+interacting with Google Cloud libraries, APIs, or services.
+* **Idiomatic.** Each sample should follow widely accepted Python best
+practices as covered below.
+
+# Sample Guidelines
+
+This section covers guidelines for Python samples. Note that
+[Testing Guidelines](#testing-guidelines) are covered separately below.
+
+## Folder Location
+
+Each sample should be in a folder under the top-level folder of
+[python-docs-samples](https://github.com/GoogleCloudPlatform/python-docs-samples)
+that corresponds to the Google Cloud service or API used by the sample.
+For example, a sample demonstrating how to work with BigTable should be
+in a subfolder under the
+[python-docs-samples/bigtable](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/bigtable)
+folder.
+
+Conceptually related samples under a service or API should be grouped into
+a subfolder. For example, App Engine Standard samples are under the
+[appengine/standard](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/standard)
```

Comment:
Should we sort based on runtime version as well? for example, `appengine/standard/python3.7`?